test/spec/fixtures/schema.sql

```diff
@@ -26,6 +26,12 @@ CREATE SCHEMA "EXTRA ""@/\#~_-";
 COMMENT ON SCHEMA v1 IS 'v1 schema';
 COMMENT ON SCHEMA v2 IS 'v2 schema';
 
+COMMENT ON SCHEMA test IS 
+$$title_test title
+
+title_test description
+that spans
+multiple lines$$;
```

Comment:
I find `title_test` a bit hard to read on COMMENT. How about:

```suggestion
COMMENT ON SCHEMA test IS 
$$My API

description
that spans
multiple lines$$;
```